doc/geany.txt

```diff
@@ -2383,6 +2383,37 @@ Always
     Always show virtual spaces beyond the end of lines
 
 
+Change History
+``````````````
+
+The change history features enables displaying of document changes in the markers margin or in the text.
+
+Newly added, modified and removed lines or words are highlighted to easily track changes to the opened
+document. The changes can be shown as vertical bars in the markers margin and/or as underlines in
+the text directly.
+
+.. note::
+    This feature may use a moderate amount of memory, especially if there are many or big changes in the document.
+    Also, the state of the modifications is not persisted, i.e. when closing the document or Geany, all change markers will be lost.
+
+
+.. image:: ./images/edit_change_history.png
+
+The image shows the default visuals:
+
+* inserted characters appear with coloured underlines
+* points where characters were deleted are shown with small triangles
+* the margin shows a block indicating the overall state of the line, prioritizing the more consequential modified states
+* the states are modified (orange), saved (green), saved then reverted to modified (green-yellow), and
+  saved then reverted to original (cyan).
+
+Show in markers margin
+    Changes are shown in the markers margin as vertical bars
+
+Show as underline indicators
+    Changes are shown as underlines in the text directly
+
```

Comment:
Maybe add "This feature is still being evaluated and is currently disabled by default."

At least anyone who RTFM will be prompted to look for a setting, not raise an issue "change history doesn't work"

Reply:
Saying that "is still being evaluated" could lead to issues asking how the status of the evaluation is. IMO this creates expectations that there is someone actively working on this. This might happen but looking back the last two years, it is not that unlikely that it won't happen.
Declaring it as "it is there, disabled and being evaluated" might cause more confusion than it tries to solve.

I think it is better to leave it as is.

Reply:
Ok. Is 85febd1 fine?

Reply:
Yes, thats why I marked resolved